central/reprocessor/reprocessor.go

```diff
@@ -339,6 +339,17 @@ func (l *loopImpl) reprocessImage(id string, fetchOpt imageEnricher.FetchOption,
 				logging.ImageName(image.GetName().GetFullName()), logging.Err(err))
 			return nil, false
 		}
+		// We need to fetch the image again to make sure all fields are populated.
+		// GetImage will internally call a Merge function which will use the CVEEdges table to enrich fields like
+		// FirstImageOccurrence and FirstSystemOccurrence.
+		image, exists, err = l.images.GetImage(allAccessCtx, id)
+		if err != nil {
+			log.Errorw("Error fetching image from database", logging.ImageID(id), logging.Err(err))
```

Comment:
Why did you use `logging.ImageID` instead of `logging.ImageName` like above?

Reply:
If there's an error `GetImage` will return a `nil` image. I think it's better to log with what we know we have for sure, the image `id`.

I can also change the code to not override the `image` variable and use the old `image` to log here.

```golang
newImage, exists, err = l.images.GetImage(allAccessCtx, id)
if err != nil {
	log.Errorw("Error fetching image from database", logging.ImageName(oldImage.GetName().GetFullName()), logging.Err(err))
	return nil, false
}
```
WDYT?

Reply:
Logging the full image name is probably more handy than the image ID as it's more human readable.

Reply:
Done